### PR TITLE
Add Flow reminder to PR template

### PR DESCRIPTION
**Pull request by jjcross**
Since CircleCI fails on Flow errors, I think it would be helpful to add a reminder to `npm run flow` before submitting a PR.

Formatting was inspired by the [Draft.js PR template](https://github.com/facebook/draft-js/blob/master/.github/PULL_REQUEST_TEMPLATE.md).

### CODE REVIEW

**Comment by gaearon** [comment]
We also need to add this step:

* If you added or removed any tests, run `./scripts/fiber/record-tests` before submitting the pull request, and commit the resulting changes.

Could you add this too, and do it both in PR template and [here](https://facebook.github.io/react/contributing/how-to-contribute.html#sending-a-pull-request)?

File: docs/contributing/how-to-contribute.md [hunk 1]
```diff
@@ -78,11 +78,12 @@ The core team is monitoring for pull requests. We will review your pull request
 
 1. Fork [the repository](https://github.com/facebook/react) and create your branch from `master`.
 2. If you've added code that should be tested, add tests!
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes (`npm test`).
-5. Make sure your code lints (`npm run lint`).
-6. Run the [Flow](https://flowtype.org/) typechecks (`npm run flow`).
-7. If you haven't already, complete the CLA.
+3. If you added or removed any tests, run ./scripts/fiber/record-tests before submitting the pull request, and commit the resulting changes.
```
Comment (gaearon) [inline docs/contributing/how-to-contribute.md]: Please enclose the command in backticks.

Comment (gaearon) [inline docs/contributing/how-to-contribute.md]: Also, this step should probably come the last before CLA. It should be done after everything else is ready.

Comment (jjcross) [inline docs/contributing/how-to-contribute.md]: That makes more sense, I've made the changes!

**Comment by gaearon** [comment]
Thanks!

**Pull request merged**